src/main/java/bio/terra/service/profile/google/GoogleBillingService.java

```diff
@@ -50,14 +50,14 @@ private static CloudBillingClient accessClient(AuthenticatedUserRequest user) {
       //  If no user, use system credentials, otherwise use user credentials instead
       final String credentialName;
       final GoogleCredentials credentials;
-      if (user == null || !user.getToken().isPresent()) {
+      if (user == null || user.getToken().isEmpty()) {
```

Comment:
wait, isn't token not an optional anymore?

Reply:
It's worth checking to see what happens if we attempt to hit an open endpoint (e.g. status or configuration)